docs/DOCKER.md

```diff
@@ -1,3 +1,63 @@
-# Telegraf Docker Images
+# Docker Images
 
-Docker images for Telegraf are kept in the [influxdata/influxdata-docker](https://github.com/influxdata/influxdata-docker/tree/master/telegraf) repo.
+Telegraf is available as an [Official image][] on DockerHub. Official images
+are a curated set of Docker Images that also automatically get security updates
+from Docker, follow a set of best practices, and are available via a shortcut
+syntax which omits the organization.
+
+InfluxData maintains Debian and Alpine based images across the last three
+minor releases. To pull the latest Telegraf images:
+
+```shell
+# latest Debian-based image
+docker pull telegraf
+# latest Alpine-based image
+docker pull telegraf:alpine
+```
+
+See the [Telegraf DockerHub][] page for complete details on available images,
+versions, and tags.
+
+[official image]: https://docs.docker.com/trusted-content/official-images/
+[Telegraf DockerHub]: https://hub.docker.com/_/telegraf
+
+## Nightly Images
+
+[Nightly builds][] are available and are generated from the master branch each
+day at around midnight UTC. The artifacts include both binary packages, RPM &
+DEB packages, as well as nightly Docker images that are hosted on [quay.io][].
+
+[Nightly builds]: /docs/NIGHTLIES.md
+[quay.io]: https://quay.io/repository/influxdb/telegraf-nightly?tab=tags&tag=latest
+
+## Dockerfiles
+
+The [Dockerfiles][] for these images are available for users to use as well.
+
+[Dockerfiles]: https://github.com/influxdata/influxdata-docker
+
+## Lockable Memory
+
+Telegraf does require the ability to use lockable memory when running. In some
```

Comment:
Maybe
```suggestion
Telegraf does require the ability to use lockable memory when running by default. In some
```
as we do have an option to avoid it...